TBD text updates for issue #22

pv0 opened the pull request:
Opinions please? :)

Changed from tables to Terminology-style format since many items
required explanation rather than specific values. Removed TBD except for
in commented-out text.

Didn't make to see how it looks in draft yet.

pv0 committed:
TBD text updates for issue #22

Opinions please? :)

Changed from tables to Terminology-style format since many items
required explanation rather than specific values. Removed TBD except for
in commented-out text.

Didn't make to see how it looks in draft yet.

pv0 committed:
Updated Appendix

Forgot to add changes to appendix on earlier commit.

Lotterleben commented on AODVv2-pandoc-edits/AODVv2.mkd:
```diff
@@ -1798,62 +1798,73 @@ MAX_SEQNUM_LIFETIME MUST be configured to have the same values for all AODVv2 ro
-*  MAX_HOPCOUNT: Routers with a value too small would not be able to discover routes to distant addresses.
-*  INFINITY_TIME: No interoperability problems due to variations on different routers, but if a
-   lower value is used, route state management may exhibit overly restrictive behavior.
+   <!-- Need to figure out what the time format. -->
```
Is this comment still valid? ^^

Lotterleben commented on AODVv2-pandoc-edits/AODVv2.mkd:
```diff
@@ -1798,62 +1798,73 @@ MAX_SEQNUM_LIFETIME MUST be configured to have the same values for all AODVv2 ro
+based on the average size of a packet containing an AODVv2 message and the bandwidth available. The limit should be 
+calculated so that AODVv2 traffic does not exceed a certain percentage of the bandwidth. The use of CONTROL_TRAFFIC_LIMIT 
+is described in [](#aodv_msgs). 
+<!-- [TBD - 50 pkts/sec?] TODO Charlie to suggest a value -->
```
@charliep51 , can you suggest a value here? :)

Lotterleben commented on AODVv2-pandoc-edits/AODVv2.mkd:
```diff
@@ -1798,62 +1798,73 @@ MAX_SEQNUM_LIFETIME MUST be configured to have the same values for all AODVv2 ro
+
+<!-- BUFFER_SIZE_PACKETS       2                         [](#route_discovery)
+BUFFER_SIZE_BYTES         MAX_PACKET_SIZE [TBD]     [](#route_discovery)
+-->
```
Imo you can delete those rather than commenting them out. The document is less cluttered this way and if we ever ned them back, we've got them in our version history. :)

Lotterleben commented on AODVv2-pandoc-edits/AODVv2.mkd:
```diff
@@ -1798,62 +1798,73 @@ MAX_SEQNUM_LIFETIME MUST be configured to have the same values for all AODVv2 ro
-Name                       Default               Description
-------                     --------              ----------
-ENABLE_IDLE_IN_RERR        Disabled              [](#RERR_gen)
+ENABLE_IDLE_IN_RERR
```
imo the table is more easy to read in this case, even though a table with one entry seems a bit silly...

pv0 replied:
I thought it was good to be consistent with the earlier sections where I changed from tables... should I re-add the table for the ENABLE_IDLE_IN_RERR in the "Network-Wide Settings" section only?

Lotterleben replied:
I'd personally be in favor of that but if @ratliffstan or @charliep51 are on your side I won't make a fuss ^^

pv0 replied:
If I don't get outvoted, it's staying the same ;) Just as a reminder, I updated a few of the tables in the Configuration section so that they look the same as the terminology section and allow a bigger description of the parameter.

pv0 committed:
moved some text in config section

Discussion of "If the values were configured differently" is now in same
order as the table above it

Lotterleben commented:
ping @charliep51 @ratliffstan for opinions :)

charliep51 commented:
Hello Lotte,

I'm not quite sure what more is needed.  We definitely don't want to 
specify a value for INFINITY_TIME.  We can specify that MAX_METRIC is 
255 for hop count.  As long as the document clearly makes strong advice 
to implement packet buffering I am O.K. to eliminate reference to the 
constants.  By the way, those buffering constants were introduced into 
the specification upon request by the usual suspects.

Regarding CONTROL_TRAFFIC_LIMIT, if you would like to include some or 
all of my text in the main body or in an appendix, I am fine.  It is 
another instance of implementation advice that has little direct impact 
on interoperability.  A poor choice might lead to zero operability, but 
that is a slightly different matter.

Regards,
Charlie P.


On 7/7/2016 10:04 AM, Lotte Steenbrink wrote:
>
> ping @charliep51 <https://github.com/charliep51> @ratliffstan 
> <https://github.com/ratliffstan> for opinions :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/Lotterleben/AODVv2-Draft/pull/26#issuecomment-231142870>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ABbdvi87KfwKA9BABTK-TlnMCSRk_KcNks5qTTGvgaJpZM4I0zt5>.
>

pv0 commented:
choose control traffic limit SHOULD not exceed 5%

charliep51 commented:
Hello folks,

On 8/3/2016 10:34 AM, pv0 wrote:

> choose control traffic limit SHOULD not exceed 5%

O.K., but use "SHOULD NOT" instead of "SHOULD not".

Regarding the measurement of CONTROL_TRAFFIC_LIMIT...

In order to produce the least perturbation of the draft, then after the 
text:

>    To avoid congestion, each AODVv2 router's rate of message generation
>    SHOULD be limited (CONTROL_TRAFFIC_LIMIT) and administratively
>    configurable.  Messages SHOULD NOT be sent more frequently than one
>    message per (1 / CONTROL_TRAFFIC_LIMIT)th of a second.

we could insert the following sentence:

> The computation of CONTROL_TRAFFIC_LIMIT is beyond the scope of this 
> specification.

If more is needed, we could make a non-normative suggestion along the 
lines of the previous discussion, but I don't think this is necessary.

Regards,
Charlie P.
